crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -618,25 +627,39 @@ en:
           everything you need to use OpenStack APIs.
       backup:
         continue: 'Continue Upgrade Process'
-        header: 'Data Backup'
+        blockui: 'Continue Upgrade Process...'
+        header: 'Database Dump'
+        download: |
+          You can download the database dump from your node <b>%{node}</b>, you can find the file at
+          <b>%{path}</b>.
```

Comment:
Could you make this nicer somehow? Now it looks

![db-dump](https://cloud.githubusercontent.com/assets/910000/12952981/d7e1e342-d018-11e5-9416-30674f183b1d.png)